portfolio/src/main/webapp/style.css

```diff
@@ -1,206 +1,201 @@
+  color: #fcf4f4;
+  margin:0 auto;
+  max-width: 800px;
 }
 
 .contact-me{
-    color: black;
-    background: #F2F2F2;
-    border-radius: 3px;
-    height: fit-content;
-    margin: 20px auto;
-    opacity: 0.7;
-    width: 500px;
+  color: black;
+  background: #F2F2F2;
+  border-radius: 3px;
+  height: fit-content;
+  margin: 20px auto;
+  opacity: 0.7;
+  width: 500px;
 }
 
 .content {
-    column-gap: 0px;
-    display: grid;
-    grid-template-columns: 550px 200px auto;
-    margin: 30% auto 0px;
+  column-gap: 0px;
+  display: grid;
+  grid-template-columns: 550px 200px auto;
+  margin: 30% auto 0px;
 }
 
 .content-body{
-    align-self: center;
-    left: 20px;
-    padding: 10px 25px;
-    position: relative;
+  align-self: center;
+  left: 20px;
+  padding: 10px 25px;
+  position: relative;
 }
 
 .content-body > h1{
-    font-size: 32px;
+  font-size: 32px;
 }
 
 .content-body .main-content-body{
-    font-size: 22px;
+  font-size: 22px;
 }
 
 .content-body > p{
-    font-size: 16px;
+  font-size: 16px;
 }
 
 .content-image{
-    align-self: center;
-    text-align: center;
+  align-self: center;
+  text-align: center;
 }
 
 .content-image img{
-    border-radius: 50%;
-    height: 200px;
-    width: 200px;
+  border-radius: 50%;
+  height: 200px;
+  width: 200px;
 }
 
 .content-image > p{
-    color: rgb(226, 241, 255) !important;
-    opacity: 0.5;
-}
-
-fieldset legend{
-    margin:0 auto;
+  color: rgb(226, 241, 255) !important;
+  opacity: 0.5;
 }
 
 form button{
-    background: rgb(39, 162, 233);
-    border: px solid black;
-    border-style: outset;
-    color: black;
-    margin-left: 10px;
-    text-shadow: none;
+  background: rgb(39, 162, 233);
+  border: px solid black;
+  border-style: outset;
+  color: black;
+  margin-left: 10px;
+  text-shadow: none;
 }
 
 .form-content{
-    align-items: flex-start;
-    display: grid;
-    grid-template-columns: 1fr 4fr;
-    padding: 5px;
+  align-items: flex-start;
+  display: grid;
+  grid-template-columns: 1fr 4fr;
+  padding: 5px;
 }
 
 .navbar{
-    display: grid;
-    font-size: 20px;
-    grid-auto-flow: column;
+  display: grid;
+  font-size: 20px;
+  grid-auto-flow: column;
 }
 
 .nav-button{
-    background-color:inherit;
-    border:none;
-    color:inherit;
-    cursor:pointer;
-    display:inline-block;
-    overflow:hidden;
-    padding:8px 16px;
-    text-align:center;
-    text-decoration:none;
-    vertical-align:middle;
-    white-space:nowrap
+  background-color:inherit;
+  border:none;
+  color:inherit;
+  cursor:pointer;
+  display:inline-block;
+  overflow:hidden;
+  padding:8px 16px;
+  text-align:center;
+  text-decoration:none;
+  vertical-align:middle;
+  white-space:nowrap
 }
 
 .nav-left-side{
-    align-self: flex-start;
+  align-self: flex-start;
 }
 
 .navbar .navbar-item{
-    border:none;
-    display:block;
-    float:left;
-    outline:0s;
-    padding:8px 16px;
-    width:auto;
+  border:none;
+  display:block;
+  float:left;
```

Comment:
Generally avoid floating things with CSS.  It can cause weird issues

Reply:
I figured that I could use the display: grid in the middle of the project, I will try to rebuild it in the coming repos